### ISSUE TITLE
Support parameter to specify target dir

### ISSUE THREAD

**Issue by Jack47** [Augusto111/STtech_test#1]
How about specify target dir in parameter? In this way we can avoid interaction when executing. So we can automate it. For example we can use it this way: `./execution_exe.py /home/ubuntu/targetDir`

**Comment by Augusto111** [comment]
OK， I‘ll do it.

**Comment by Augusto111** [comment]
yeah, my input method is too "freshman", and didn't think about automation.

**Comment by Augusto111** [comment]
yeah, my input method is too "freshman", and didn't think about automation.
